src/lib/Behat/PageElement/ContentUpdateForm.php

```diff
@@ -25,9 +25,9 @@ public function __construct(BrowserContext $context)
         parent::__construct($context);
         $this->fields = [
             'formElement' => '[name=ezplatform_content_forms_content_edit],[name=ezplatform_content_forms_user_create],[name=ezplatform_content_forms_user_update]',
-            'closeButton' => '.ez-content-edit-container__close',
+            'closeButton' => '.ez-content-edit-container__close, .ibexa-content-edit-sidebar__header .ibexa-content-edit-sidebar__back',  // TO DO: Set one selector after redesign
```

Comment:
TODO? Here and in other places.

Reply:
> TODO? Here and in other places.

yes, because after whole redesign we have to change it